### PR TITLE
fix ineffectual assignment

### PR DESCRIPTION
**Pull request by JieJhih**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[X] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: security/pkg/nodeagent/cache/secretcache.go [hunk 1]
```diff
@@ -292,8 +292,8 @@ func (sc *SecretCache) addFileWatcher(file string, token string, connKey ConnKey
 	// checking for duplicates. This check is needed here to avoid processing duplicate events for the same file.
 	sc.certMutex.Lock()
 	npath := filepath.Clean(file)
-	watching := false
-	if _, watching = sc.fileCerts[npath]; !watching {
+	_, watching := sc.fileCerts[npath]
```
Comment (ramaraochavali) [inline security/pkg/nodeagent/cache/secretcache.go:295]: Sorry, I did not understand, what is the problem with the above?

Comment (ramaraochavali) [reply security/pkg/nodeagent/cache/secretcache.go:295]: You mean changing it immediately and line `295` is ineffectual?

File: security/pkg/nodeagent/cache/secretcache.go [hunk 2]
```diff
@@ -292,8 +292,8 @@ func (sc *SecretCache) addFileWatcher(file string, token string, connKey ConnKey
 	// checking for duplicates. This check is needed here to avoid processing duplicate events for the same file.
 	sc.certMutex.Lock()
 	npath := filepath.Clean(file)
-	watching := false
-	if _, watching = sc.fileCerts[npath]; !watching {
```
Comment (howardjohn) [inline security/pkg/nodeagent/cache/secretcache.go:296]:
```suggestion
	if _, watching := sc.fileCerts[npath]; !watching {	
```

why not just do this?

**Comment by howardjohn** [comment]
Please see https://github.com/istio/istio/wiki/Development-Conventions#superfluous-changes. Thanks!

File: security/pkg/nodeagent/cache/secretcache.go (hunk 2 above)
Comment (JieJhih) [reply security/pkg/nodeagent/cache/secretcache.go:296]: @howardjohn This is followed https://goreportcard.com/report/github.com/istio/istio#ineffassign. The two variables changed in this PR, the two variables have been assigned value is from:
https://github.com/istio/istio/blob/master/security/pkg/nodeagent/cache/secretcache.go#L296
https://github.com/istio/istio/blob/master/security/pkg/nodeagent/cache/secretcache.go#L803
It seems to me that this PR can help the code cleaner.

**Pull request closed without merging**